### DISCUSSION TITLE
Dropdown Issue in Version 10.5.1

### DISCUSSION THREAD

**Discussion by girivelugula47** [primefaces/community#1810] [PrimeReact] [answered]
![Lincoln-Learning-Solutions](https://github.com/primefaces/community/assets/168617823/4c54b2e9-661a-4d5a-ae9b-a3ebd57d3d88)

Here when I use the keyboard while selecting values, the scrolling is not working as move down using the keyboard down areas. Please resolve this issue as soon as possible,
Version : 10.5.1
Thank you.

**Comment by melloware** [comment, marked as answer]
Try 10.6.4
